Wrappers/Python/ccpi/optimisation/operators/GradientOperator.py

```diff
@@ -396,11 +367,11 @@ def direct(self, x, out=None):
         arg2 = [el for el in x.shape]
         args = arg1 + arg2 + [self.bnd_cond, 1, self.num_threads]
         self.fd(x_p, *args)
+        out /= self.voxel_size_order
```

Comment:
This should skip the divide if voxel size equals 1

Reply:
done